Dockerfile.fedora

```diff
@@ -2,22 +2,19 @@ FROM fedora
 MAINTAINER langdon <langdon@fedoraproject.org>
 #Derived from Vaclav Pavlin <vpavlin@redhat.com>; https://github.com/vpavlin/atomicapp-run
 
-RUN yum -y update && yum clean all
-RUN yum -y install python python-pip docker && yum clean all
-
-ADD requirements.txt /opt/atomicapp/
-ADD setup.py /opt/atomicapp/
-ADD atomicapp/ /opt/atomicapp/atomicapp/
-
-WORKDIR /opt/atomicapp
-RUN python setup.py install
+RUN yum -y update
+RUN yum -y install python python-pip docker-io kubernetes && yum clean all
 
+COPY . /opt/atomicapp/
+WORKDIR /opt/atomicapp/
+RUN chmod u+x /opt/atomicapp/setup.py
+RUN ./setup.py install
 
 WORKDIR /application-entity
-VOLUME /application-entity/answers.conf
-
-LABEL RUN docker run -it --rm --privileged --net=host -v ${PWD}:/atomicapp -v /run:/run -v /:/host -v ${PWD}/answers.conf:/application-entity/answers.conf --name NAME -e NAME=NAME -e IMAGE=IMAGE IMAGE atomicapp -v run /atomicapp
-LABEL INSTALL docker run --rm -it --privileged -v /run:/run -v ${PWD}:/atomicapp -v /:/host -v ${PWD}/answers.conf:/application-entity/answers.conf -e IMAGE=IMAGE -e NAME=NAME --name NAME IMAGE atomicapp -v install --path /atomicapp /application-entity
+VOLUME /application-entity
 
-CMD  atomicapp -h
+LABEL RUN docker run --rm -it --privileged --net=host -v ${PWD}:/atomicapp -v /run:/run -v /:/host -v ${PWD}:/application-entity --name NAME -e NAME=NAME -e IMAGE=IMAGE IMAGE atomicapp -v run /atomicapp
```

Comment:
This is broken - with `-v ${PWD}:/application-entity` you would replace the app content with the content of `${PWD}`